pages/clubs-toolkit.jsx

```diff
@@ -46,6 +46,17 @@ var ToolkitList = React.createClass({
 
 var ToolkitListItem = React.createClass({
   render: function() {
+    if (this.props.headerHref) {
+      return (
+        <li>
+          <p>
+            <span className="italic"><a href={this.props.headerHref}>{this.props.header}</a> </span>
+            <span>{this.props.children}</span>
```

Comment:
I wonder if there is a way to not need to copy and past this fairly similar block?